cloud-haven-react-app/src/components/components.js

```diff
@@ -1,2 +1,3 @@
 export {default as Home} from './Home/Home';
-export {default as Test} from './Test';
\ No newline at end of file
+export {default as Test} from './Test';
+export {default as SignUp} from './SignUp/SignUp';
```

Comment:
```suggestion
export {default as SignUp} from './SignUp/SignUp';

```
GitHub I think requires a newline at the end of files. Looks like I had originally left a newline off (oops, my bad), but maybe we could add one in with this PR?